app/client/src/config/fields.ts

```diff
@@ -268,7 +268,12 @@ export const filterFields = [
     label: 'Parameter IR Category',
     type: 'multiselect',
   },
-  { key: 'parameterName', label: 'Parameter Name', type: 'multiselect' },
+  {
+    key: 'parameterName',
+    label: 'Parameter Name',
+    type: 'multiselect',
+    tooltip: 'The dropdown list only contains active (non-retired) parameter names.',
```

Comment:
This is a minor point, but I think it would be more correct to phrase this as "The dropdown list contains only active (non-retired) parameter names." (different placement of _only_). I went ahead and committed this change.